backend/src/main/java/ch/nova_omnia/lernello/service/LearningKitService.java

```diff
@@ -1,28 +1,32 @@
 package ch.nova_omnia.lernello.service;
 
-import java.util.List;
-import java.util.Optional;
-import java.util.UUID;
-
-import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
-
 import ch.nova_omnia.lernello.model.data.LearningKit;
-import ch.nova_omnia.lernello.repository.FileRepository;
 import ch.nova_omnia.lernello.repository.LearningKitRepository;
-import ch.nova_omnia.lernello.repository.UserRepository;
 import jakarta.persistence.EntityNotFoundException;
 import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
 
 @Service
 @RequiredArgsConstructor
 public class LearningKitService {
     private final LearningKitRepository learningKitRepository;
-    private final UserRepository userRepository;
-    private final FileRepository fileRepository;
 
     public List<LearningKit> findAll() {
-        return learningKitRepository.findAll();
+        return learningKitRepository.findAll().stream()
+            .sorted((kit1, kit2) -> kit2.getCreatedAt().compareTo(kit1.getCreatedAt()))
+            .toList();
+    }
+
+    public List<LearningKit> findLatestFive() {
+        return learningKitRepository.findAll().stream()
```

Comment:
This is performance wise not scalable. Sorting and limiting is something that DB should do via a query.
The `learningKitRepository` should provide some default methods for this already .

Again, this works, and for the sake of the prototype Im okay with merging it